src/solidity/types/Address.js

```diff
@@ -12,4 +12,18 @@ Address.prototype.decodeFromStorage = function (location, storageContent) {
   return '0x' + value.toUpperCase()
 }
 
+Address.prototype.decodeLocals = function (stackHeight, stack, memory) {
+  if (stack.length - 1 < stackHeight) {
+    return '0x0000000000000000000000000000000000000000'
+  } else {
+    return '0x' + util.extractHexByteSlice(stack[stack.length - 1 - stackHeight], this.storageBytes, 0)
+  }
+}
+
+Address.prototype.decodeFromMemory = function (offset, memory) {
```

Comment:
Both this and the preceding function seems to share a lot of code with the respective functions from other types, please try to extract the common part.